docs/diagrams/FindSequenceDiagram.puml

```diff
@@ -0,0 +1,73 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":FindCommandParser" as FindCommandParser LOGIC_COLOR
+participant ":ParserUtil" as ParserUtil LOGIC_COLOR
+participant ":NameContainsKeywordPredicate" as NameContainsKeywordPredicate LOGIC_COLOR
+participant "d:FindCommand" as FindCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("find n/Lucy n/Adam")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("find n/Lucy n/Adam")
+activate AddressBookParser
+
+create FindCommandParser
+AddressBookParser -> FindCommandParser
+activate FindCommandParser
+
+FindCommandParser --> AddressBookParser
+deactivate FindCommandParser
+
+AddressBookParser -> FindCommandParser : parse("n/Lucy n/Adam")
+activate FindCommandParser
+
+FindCommandParser -> ParserUtil
+activate ParserUtil
+return keywords
+
+FindCommandParser -> NameContainsKeywordPredicate : keywords
+activate NameContainsKeywordPredicate
+return predicate
+
+create FindCommand
+FindCommandParser -> FindCommand : predicate
```

Comment:
I don't know if you can pass the variable like that in a sequence diagram? I think you might need to show it is a method call so something like FindCommand(predicate) instead?